point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.abmedge.frontend;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class FrontendController {
+  private static final Logger LOGGER =
+      LogManager.getLogger(FrontendController.class);
+
+  /**
+   * Readiness probe endpoint.
+   *
+   * @return HTTP Status 200 if server is ready to receive requests.
+   */
+  @GetMapping("/ready")
+  @ResponseStatus(HttpStatus.OK)
+  public String readiness() {
+    return "ok";
+  }
+
+  /**
+   * Liveness probe endpoint.
+   *
+   * @return HTTP Status 200 if server is healthy and serving requests.
+   */
+  @GetMapping("/healthy")
+  public ResponseEntity<String> liveness() {
+    // TODO:: Add suitable liveness check
+    return new ResponseEntity<>("ok", HttpStatus.OK);
+  }
+
+  @RequestMapping("/")
+  public String home() {
```

Comment:
What does this home do?

Reply:
Since this PR only introduces the stubs for all the services the `/` context just prints out a welcome message: `"Hello Anthos BareMetal - Frontend Controller"`. But in the final version when all is implemented the `/` context directly takes you to the POS app